app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -153,25 +133,24 @@
 
     private AndroidCommCarePlatform platform;
 
-    private SquareButtonWithNotification startButton;
-    private SquareButtonWithNotification logoutButton;
-    private SquareButtonWithNotification viewIncomplete;
-    private SquareButtonWithNotification syncButton;
-
-    private HomeScreenAdapter adapter;
-    private GridViewWithHeaderAndFooter gridView;
-    private StaggeredGridView newGridView;
-    private View mTopBanner;
-
+    private HomeActivityUIController uiController;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        finishIfNotRoot();
```

Comment:
this used to bail after this call if it wasn't the root, does returning v. not returning from onCreate() in that case affect anything?